_episodes/03-create.md

```diff
@@ -18,6 +18,9 @@ We will continue with the story of Wolfman and Dracula who are investigating if
 is possible to send a planetary lander to Mars. 
 
 ![motivatingexample](../fig/motivatingexample.png)
+[Werewolf vs dracula](https://www.deviantart.com/b-maze/art/Werewolf-vs-Dracula-124893530): [Deviant Art](https://www.deviantart.com/) / [b-maze](https://www.deviantart.com/b-maze). [Planets](https://commons.wikimedia.org/wiki/File:Planets_are_us.png), [Mummy](https://commons.wikimedia.org/wiki/File:Mummy_icon_-_Noun_Project_4070.svg) (&copy; [TintoMeches](https://commons.wikimedia.org/wiki/User_talk:TintoMeches)),
```

Comment:
Would you mind adding line breaks in here so that things are <100 characters? I think if you add newlines after each link the final formatting should be preserved.